### DISCUSSION TITLE
What is an "Invalid" package?

### DISCUSSION THREAD

**Discussion by MGatner** [composer-unused/composer-unused#330] [Q&A] [answered]
I found references in the code but the docs don't explain what makes a package "invalid".

In [this case](https://github.com/tattersoftware/codeigniter4-tools/runs/5362052564?check_suite_focus=true#step:7:9) the package is the only dependency but because the files are only referenced from a Bash script I have to ignore the package with an `exclude` statement. This causes it to end up as a "warning" instead of an ignored dependency.

**Comment by icanhazstring** [comment, marked as answer]
This comes from this line:
https://github.com/composer-unused/composer-unused/blob/main/src/OutputFormatter/GithubFormatter.php#L56

It is an ignored package. So the result is correct, but the text is kind of misleading. 

Maybe a warning is not that good and should just be an info with better text 😉

**Reply by MGatner** [reply]
Gotcha! So just confirming, this should *not* denote a failure right?

**Reply by icanhazstring** [reply]
Correct. This is just an info. 

Ignored packages, same for every output, are not counted towards an exit code > 0
